app/src/main/res/values/strings.xml

```diff
@@ -1287,15 +1287,15 @@
     <string name="direction_not_set">Direction not set</string>
     <!--The compass direction / bearing-->
     <string name="direction">Direction</string>
-    <string name="augmented_reality">Augmented reality</string>
+    <string name="augmented_reality">Augmented Reality</string>
     <string name="augmented_reality_description">View nearby beacons and the sun / moon with your camera</string>
     <string name="magnetic_north">Magnetic North</string>
```

Comment:
Can you update the magnetic_north string to be sentenced cased as well, to match true_north?